### ISSUE TITLE
Add article to `init/cold_storage.md`

### ISSUE THREAD

**Issue by simonsan** [rustic-rs/docs#105]
Here some documentation essentially, how to migrate to a cold storage:
https://kmh.prasil.info/posts/rustic-cold-storage-glacier-migration-configuration/

**Comment by aawsome** [comment]
see https://rustic.cli.rs/docs/stories.html

However, as this uses the old config syntax for `[copy]` we should maybe add a note about how to adapt this to rustic 0.9.0.

**Comment by simonsan** [comment]
> see https://rustic.cli.rs/docs/stories.html
> 
> However, as this uses the old config syntax for `[copy]` we should maybe add a note about how to adapt this to rustic 0.9.0.

Agreed, but I think the article is not really helpful where it is. It's less a story about rustic, than some `how to`, so I think we need to link to it directly from where it's useful. Also, we should archive the links and link to the archived ones, as they are not within our control and could be outdated or being replaced with content we don't support.
